SEP-0002-revised.md

```diff
@@ -0,0 +1,245 @@
+This diversity should consider things such as research topic, career stage, as well as all other forms of diversity such as gender.
+
+* **Removal**: A board member may be removed by a two-thirds majority vote of the full board, if before any meeting of the board at which a vote on removal will be made the member in question is given electronic or written notification of the board’s intention to discuss their case and is given the opportunity to be heard at a meeting of the board.
+Alternatively a member of the board can be removed by a vote of the Voting Members, following the same procedure laid out below for the removal of a member of the Steering Committee.
+
+* **Meetings**:  The board shall have a meeting at least every quarter at times and places fixed by the board.
+Board meetings shall provide at least one week (7 days) notice to all board members and the Steering Committee.
+Notice of meetings shall specify the place, day, and hour of meeting.
+The purpose of the meeting needs not be specified.
+
+* **Board officers**: The officers of the board shall be a board chair, vice-chair, and secretary, all of whom shall be chosen by, and serve at the pleasure of, the board.
+Officers shall serve one year terms (as officers) and can serve consecutive terms.
+
+* **Board Chair**: The board chair shall lead the board in performing its duties and responsibilities, including, if present, leading all meetings of the board.
+
+* **Vice-chair**: In the absence of the board chair, the vice-chair shall perform the duties of the board chair.
+When so acting, the vice-chair shall have all the powers of and be subject to all the restrictions upon the board chair.
+
+* **Secretary**: The secretary shall be responsible for keeping the minutes of all meetings and actions of the board.
+The minutes of each meeting shall state the time and place that it was held, the members present, and such other information as shall be necessary to determine the actions taken.
+The minutes of all board meetings shall be made publicly accessible to the whole SunPy community.
+
+* **Advise and Consent**: The steering committee shall present the state of the project and future plans at meetings of the board.
+The advisory board shall advise and consent on those plans.
+
+
+## The Steering Committee
+The role of the steering committee, which is headed by the Lead Developer, is to manage the day to day operations of the SunPy project.
+The steering committee shall be responsible for the running of the project under advisement of the board.
+This includes:
+
+* holding regular meetings with the developer community (at least monthly)
+* making regular reports to the board (at least quarterly)
+* having ownership privileges to the SunPy GitHub organization
+* creating, maintaining and enforcing the SunPy code of conduct (see below)
+* accepting or rejecting SEPs based on community consensus
```

Comment:
This is a bit awkward because it allows for the steering committee to edit this very SEP unilaterally and remove the board without input from the board. How can the board "ensure that the governance of the project is running as intended." if the steering committee could just delete its governance structure. I know that it mentions that the community is involved in SEPs but maybe we should elevate a few SEPs as privileged and requiring the board to approve?

Reply:
I think the escape hatch there is "...based on community consensus" but it's not clear what exactly that means.

Reply:
But is the board part of the community? It could have people outside of the normal community.

Reply:
I think we probably need to do what Astropy did and exclude this SEP from the usual process. APE 0 can only be amended by a vote of the voting members.